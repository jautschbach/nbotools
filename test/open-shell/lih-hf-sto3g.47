 $GENNBO  NATOMS=2  NBAS=6  OPEN  UPPER  BODM  FORMAT=PRECISE  $END
$NBO plot SAO  SPNAO FAO=w SPNBO=W  NAONHO=W NHONBO=W NBONLMO=W NLMOMO=W FILE=lih-hf-sto3g  $END
 $COORD
 ORCA Job: lih-hf-sto3g
     3    3       1.148233      -0.071863       0.014091
     1    1      -0.381767      -0.071863       0.014091
 $END
 $BASIS
  CENTER =     1     1     1     1     1     2
   LABEL =     1     1   103   101   102     1
 $END
 $CONTRACT
  NSHELL =     4
    NEXP =    12
   NCOMP =     1     1     3     1
   NPRIM =     3     3     3     3
    NPTR =     1     4     7    10
     EXP =  0.161195750000E+02  0.293620070000E+01  0.794650500000E+00
            0.636289700000E+00  0.147860100000E+00  0.480887000000E-01
            0.636289700000E+00  0.147860100000E+00  0.480887000000E-01
            0.342525091000E+01  0.623913730000E+00  0.168855400000E+00
      CS =  0.884856287789E+00  0.855794457874E+00  0.266713752926E+00
           -0.507585163759E-01  0.678936361973E-01  0.512403447492E-01
            0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
            0.276934360953E+00  0.267838851753E+00  0.834736696169E-01
      CP =  0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
            0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
            0.126299062547E+00  0.794202811582E-01  0.125815088415E-01
            0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
 $END
 $OVERLAP
  0.100000000000E+01  0.241136651184E+00  0.100000000000E+01 -0.625934863094E-18
  0.000000000000E+00  0.100000000000E+01 -0.490875369635E-16 -0.178020483580E-15
  0.000000000000E+00  0.100000000000E+01  0.161839381764E-17  0.101130248225E-17
  0.000000000000E+00 -0.198293399139E-32  0.100000000000E+01  0.771982236976E-01
  0.412771702639E+00 -0.296575867518E-18 -0.525321457201E+00 -0.755399575866E-18
  0.100000000000E+01
 $END
 $DENSITY
  0.999049365173E+00  0.912437135711E-02  0.269195304970E-01 -0.868339048937E-15
 -0.953070489662E-16  0.103922078703E-29  0.232154205638E-01 -0.308962936817E-01
  0.811081107095E-16  0.366000047902E-01  0.666770273978E-16 -0.165312075276E-15
  0.482271970925E-30  0.193883851563E-15  0.103029553784E-29 -0.941973973276E-01
  0.132655661842E+00 -0.352843968936E-15 -0.156959640652E+00 -0.831780376173E-15
  0.673152861230E+00
  0.984506592019E+00  0.299002170672E-01  0.908092427126E-03 -0.776393843906E-15
 -0.235796739714E-16  0.612273605624E-30 -0.672740812013E-03 -0.204316522329E-04
  0.530531567006E-18  0.459702559451E-06 -0.356303727425E-15 -0.108212163110E-16
  0.280985442633E-30  0.243472274187E-18  0.128950224616E-30  0.909826059252E-03
  0.276321122536E-04 -0.717499869635E-18 -0.621709521148E-06 -0.329276023998E-18
  0.840810477862E-06
 $END
 $FOCK
 -0.273126441189E+01 -0.730173143542E+00 -0.472832726133E+00  0.202017508088E-15
  0.620138145412E-16 -0.104007991494E+00  0.219110402306E-02  0.157186269808E+00
 -0.478739147084E-16 -0.313149827755E+00  0.605252456994E-16  0.111430562101E-15
 -0.962513989069E-31 -0.136840551602E-15 -0.104007991494E+00 -0.244129473610E+00
 -0.449807358014E+00  0.857164136907E-16  0.518296832166E+00  0.216382503820E-15
 -0.804668978101E+00
 -0.272364443513E+01 -0.729518610891E+00 -0.375467352795E+00  0.213611627210E-15
  0.203216722130E-16 -0.927529079443E-01  0.218117548264E-02  0.247319972747E-01
  0.447321712330E-17 -0.129496155194E+00 -0.301902814615E-16 -0.441800029497E-16
  0.161300063155E-31  0.133261369997E-16 -0.927529079443E-01 -0.241490380101E+00
 -0.210334067857E+00  0.805462267148E-17  0.178762794607E+00 -0.170777905388E-16
 -0.109508155214E+00
 $END
 $LCAOMO
  0.991950064508E+00  0.291889656088E-01 -0.927708368336E-15 -0.283438942847E-03
 -0.582516890904E-16  0.662032032472E-02  0.122818706618E+00 -0.161454435627E+00
  0.422584867630E-15  0.191311067251E+00  0.101336186950E-14 -0.820432223032E+00
 -0.234848322037E+00  0.931642458682E+00 -0.222831994611E-14  0.445577140948E+00
  0.336406571385E-14 -0.169585668042E+00 -0.406085992269E-15  0.979411543656E-15
 -0.580951480144E+00  0.527603652433E-15 -0.813938190355E+00 -0.446833753202E-15
 -0.437243137132E-15  0.278026919875E-14  0.813938190355E+00  0.125767684098E-14
 -0.580951480144E+00 -0.561552822746E-15 -0.924681443605E-01  0.695330776475E+00
 -0.701636529033E-16 -0.112417726929E+01 -0.765956405579E-16 -0.105166073827E+01
 -0.992223055577E+00 -0.301345719586E-01  0.782479140696E-15  0.678013686773E-03
  0.359096400172E-15 -0.916957184312E-03 -0.213484252806E+00  0.665617606172E+00
  0.125774752970E-15 -0.343662972192E+00 -0.233062055908E-15  0.370118167714E+00
 -0.167485745791E+00  0.691919210490E+00  0.255229000605E-14  0.686564435426E+00
  0.324055198221E-14 -0.204338687327E+00 -0.542011301333E-15  0.172413087770E-14
 -0.963729554431E+00  0.193158788494E-14 -0.266880770975E+00 -0.539080238707E-15
  0.194888648553E-15 -0.123326081981E-14 -0.266880770975E+00 -0.147155219943E-14
  0.963729554431E+00  0.447215808474E-15 -0.679695814292E-01  0.675022226383E+00
 -0.392204649885E-16 -0.953644642394E+00 -0.658614990500E-16 -0.127638547094E+01
 $END
 $KINETIC
  0.357678642449E+01 -0.202373805613E-01  0.102163366117E+00  0.224155886469E-18
  0.000000000000E+00  0.319681669225E+00 -0.405824479811E-16 -0.426821941514E-16
  0.000000000000E+00  0.319681669225E+00 -0.524063198095E-17  0.895393127669E-18
  0.000000000000E+00 -0.110461960951E-33  0.319681669225E+00 -0.167562356894E-01
  0.469315040394E-01  0.950447108663E-19 -0.139729321775E+00 -0.651667393243E-18
  0.760031883567E+00
 $END
 $NUCLEAR
 -0.832874881867E+01 -0.104773582571E+01 -0.150578768539E+01  0.222656515657E-17
  0.000000000000E+00 -0.146212120661E+01  0.175981856564E-01  0.125931970900E+00
  0.000000000000E+00 -0.156335787944E+01 -0.835734626835E-17 -0.234410857709E-17
  0.000000000000E+00  0.262894143351E-18 -0.146212120661E+01 -0.335239398727E+00
 -0.784550376331E+00  0.545106434743E-18  0.101277659836E+01  0.438319768214E-18
 -0.226052662359E+01
 $END
