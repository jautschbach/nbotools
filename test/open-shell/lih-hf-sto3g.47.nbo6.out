
 *********************************** NBO 6.0 ***********************************
             N A T U R A L   A T O M I C   O R B I T A L   A N D
          N A T U R A L   B O N D   O R B I T A L   A N A L Y S I S
 *************************** Erik Van Lenthe (100424) **************************
  (c) Copyright 1996-2017 Board of Regents of the University of Wisconsin System
      on behalf of the Theoretical Chemistry Institute.  All rights reserved.

          Cite this program as:

          NBO 6.0.  E. D. Glendening, J. K. Badenhoop, A. E. Reed,
          J. E. Carpenter, J. A. Bohmann, C. M. Morales, C. R. Landis,
          and F. Weinhold (Theoretical Chemistry Institute, University
          of Wisconsin, Madison, WI, 2013); http://nbo6.chem.wisc.edu/

       /NAONHO / : Write the NAO to NHO transformation to lfn49
       /NHONBO / : Write the NHO to NBO transformation to lfn49
       /NBONLMO/ : Write the NBO to NLMO transformation to lfn49
       /NLMOMO / : Write the NLMO to MO transformation to lfn49
       /FAO    / : Write the AO Fock matrix to lfn49
       /SAO    / : Print the AO overlap matrix
       /SPNAO  / : Print the PNAO overlap matrix
       /SPNBO  / : Write the PNBO overlap matrix to lfn49
       /PLOT   / : Write information for the orbital plotter
       /FILE   / : Set to lih-hf-sto3g

 Filename set to lih-hf-sto3g

 Job title:  ORCA Job: lih-hf-sto3g


 AO overlap matrix:

          AO         1       2       3       4       5       6
      ---------- ------- ------- ------- ------- ------- -------
   1. Li 1 (s)    1.0000  0.2411 -0.0000 -0.0000  0.0000  0.0772
   2. Li 1 (s)    0.2411  1.0000  0.0000 -0.0000  0.0000  0.4128
   3. Li 1 (pz)  -0.0000  0.0000  1.0000  0.0000  0.0000 -0.0000
   4. Li 1 (px)  -0.0000 -0.0000  0.0000  1.0000 -0.0000 -0.5253
   5. Li 1 (py)   0.0000  0.0000  0.0000 -0.0000  1.0000 -0.0000
   6.  H 2 (s)    0.0772  0.4128 -0.0000 -0.5253 -0.0000  1.0000


 NATURAL POPULATIONS:  Natural atomic orbital occupancies

  NAO Atom No lang   Type(AO)    Occupancy     Spin
 -----------------------------------------------------
   1   Li  1  s      Cor( 1s)     2.00000     0.00000
   2   Li  1  s      Val( 2s)     0.03258     0.03258
   3   Li  1  px     Ryd( 2p)     0.02442     0.02442
   4   Li  1  py     Ryd( 2p)     0.00000     0.00000
   5   Li  1  pz     Ryd( 2p)     0.00000     0.00000

   6    H  2  s      Val( 1s)     0.94300     0.94300


 Summary of Natural Population Analysis:

                                     Natural Population                 Natural
             Natural    ---------------------------------------------    Spin
  Atom No    Charge        Core      Valence    Rydberg      Total      Density
 -------------------------------------------------------------------------------
   Li  1    0.94300      2.00000     0.03258    0.02442     2.05700     0.05700
    H  2    0.05700      0.00000     0.94300    0.00000     0.94300     0.94300
 ===============================================================================
 * Total *  1.00000      2.00000     0.97559    0.02442     3.00000     1.00000

                                 Natural Population
 ---------------------------------------------------------
   Core                       2.00000 ( 99.9998% of    2)
   Valence                    0.97559 ( 97.5586% of    1)
   Natural Minimal Basis      2.97558 ( 99.1861% of    3)
   Natural Rydberg Basis      0.02442 (  0.8139% of    3)
 ---------------------------------------------------------

    Atom No         Natural Electron Configuration
 ----------------------------------------------------------------------------
     Li  1      [core]2s( 0.03)2p( 0.02)
      H  2            1s( 0.94)


 PNAO overlap matrix:

          NAO        1       2       3       4       5       6
      ---------- ------- ------- ------- ------- ------- -------
   1. Li 1( 1s )  1.0000 -0.0000 -0.0000  0.0000 -0.0000  0.0888
   2. Li 1( 2s ) -0.0000  1.0000 -0.0000  0.0000  0.0000  0.4038
   3. Li 1( 2px) -0.0000 -0.0000  1.0000 -0.0000  0.0000 -0.5253
   4. Li 1( 2py)  0.0000  0.0000 -0.0000  1.0000  0.0000 -0.0000
   5. Li 1( 2pz) -0.0000  0.0000  0.0000  0.0000  1.0000 -0.0000
   6.  H 2( 1s )  0.0888  0.4038 -0.5253 -0.0000 -0.0000  1.0000


 ***************************************************
 *******         Alpha spin orbitals         *******
 ***************************************************


 NATURAL POPULATIONS:  Natural atomic orbital occupancies

  NAO Atom No lang   Type(AO)    Occupancy      Energy
 -------------------------------------------------------
   1   Li  1  s      Cor( 1s)     1.00000      -2.73349
   2   Li  1  s      Val( 2s)     0.03258      -0.14046
   3   Li  1  px     Ryd( 2p)     0.02442       0.08358
   4   Li  1  py     Ryd( 2p)     0.00000      -0.10401
   5   Li  1  pz     Ryd( 2p)     0.00000      -0.10401

   6    H  2  s      Val( 1s)     0.94300      -0.76875


 Summary of Natural Population Analysis:

                                     Natural Population
             Natural    ---------------------------------------------
  Atom No    Charge        Core      Valence    Rydberg      Total
 --------------------------------------------------------------------
   Li  1    0.44300      1.00000     0.03258    0.02442     1.05700
    H  2   -0.44300      0.00000     0.94300    0.00000     0.94300
 ====================================================================
 * Total *  0.00000      1.00000     0.97558    0.02442     2.00000

                                 Natural Population
 ---------------------------------------------------------
   Core                       1.00000 ( 99.9998% of    1)
   Valence                    0.97558 ( 97.5584% of    1)
   Natural Minimal Basis      1.97558 ( 98.7791% of    2)
   Natural Rydberg Basis      0.02442 (  1.2209% of    2)
 ---------------------------------------------------------

    Atom No         Natural Electron Configuration
 ----------------------------------------------------------------------------
     Li  1      [core]2s( 0.03)2p( 0.02)
      H  2            1s( 0.94)


 NATURAL BOND ORBITAL ANALYSIS, alpha spin orbitals:

                            Occupancies       Lewis Structure    Low   High
         Max    Occ     -------------------  -----------------   occ   occ
  Cycle  Ctr   Thresh    Lewis   non-Lewis     CR  BD  nC  LP    (L)   (NL)
 ============================================================================
    1     2     0.95     2.00000   0.00000      1   1   0   0     0      0
 ----------------------------------------------------------------------------

 Structure accepted: No low occupancy Lewis orbitals

 -------------------------------------------------------
   Core                      1.00000 (100.000% of   1)
   Valence Lewis             1.00000 (100.000% of   1)
  ==================      =============================
   Total Lewis               2.00000 (100.000% of   2)
  -----------------------------------------------------
   Valence non-Lewis         0.00000 (  0.000% of   2)
   Rydberg non-Lewis         0.00000 (  0.000% of   2)
  ==================      =============================
   Total non-Lewis           0.00000 (  0.000% of   2)
 -------------------------------------------------------


     (Occupancy)   Bond orbital / Coefficients / Hybrids
 ------------------ Lewis ------------------------------------------------------
   1. (1.00000) CR ( 1)Li  1            s(100.00%)
                                         1.0000  0.0000  0.0000  0.0000  0.0000
   2. (1.00000) BD ( 1)Li  1- H  2
               (  5.70%)   0.2387*Li  1 s( 57.16%)p 0.75( 42.84%)
                                         0.0000  0.7561 -0.6545 -0.0000 -0.0000
               ( 94.30%)   0.9711* H  2 s(100.00%)
                                         1.0000
 ---------------- non-Lewis ----------------------------------------------------
   3. (0.00000) BD*( 1)Li  1- H  2
               ( 94.30%)   0.9711*Li  1 s( 57.16%)p 0.75( 42.84%)
               (  5.70%)  -0.2387* H  2 s(100.00%)
   4. (0.00000) RY ( 1)Li  1            s(  0.00%)p 1.00(100.00%)
   5. (0.00000) RY ( 2)Li  1            s(  0.00%)p 1.00(100.00%)
   6. (0.00000) RY ( 3)Li  1            s( 42.84%)p 1.33( 57.16%)


 NHO DIRECTIONALITY AND BOND BENDING (deviation from line of nuclear centers at
                                      the position of maximum hybrid amplitude)

         [Thresholds for printing:  angular deviation  >  1.0 degree]
                                    p- or d-character  > 25.0%
                                    orbital occupancy  >  0.10e

                        Line of Centers        Hybrid 1             Hybrid 2
                        ---------------  -------------------  ------------------
           NBO            Theta   Phi    Theta   Phi    Dev   Theta   Phi    Dev
 ===============================================================================
 Failed to locate density maximum for hybrid 2 on center 1


 SECOND ORDER PERTURBATION THEORY ANALYSIS OF FOCK MATRIX IN NBO BASIS

     Threshold for printing:   0.25 kcal/mol
                                                          E(2) E(NL)-E(L) F(L,NL)
      Donor (L) NBO              Acceptor (NL) NBO      kcal/mol   a.u.    a.u.
 ===============================================================================

 within unit  1
       None above threshold


 NATURAL BOND ORBITALS (Summary):

                                                     Principal Delocalizations
           NBO                 Occupancy    Energy   (geminal,vicinal,remote)
 ===============================================================================
 Molecular unit  1  (HLi)
 ------ Lewis --------------------------------------
    1. CR ( 1)Li  1             1.00000    -2.73349
    2. BD ( 1)Li  1- H  2       1.00000    -0.82001
 ------ non-Lewis ----------------------------------
    3. BD*( 1)Li  1- H  2       0.00000     0.07944
    4. RY ( 1)Li  1             0.00000    -0.10401
    5. RY ( 2)Li  1             0.00000    -0.10401
    6. RY ( 3)Li  1             0.00000    -0.08506
          -------------------------------
                 Total Lewis    2.00000  ( 99.9999%)
           Valence non-Lewis    0.00000  (  0.0001%)
           Rydberg non-Lewis    0.00000  (  0.0000%)
          -------------------------------
               Total unit  1    2.00000  (100.0000%)
              Charge unit  1    0.00000


 NATURAL LOCALIZED MOLECULAR ORBITAL (NLMO) ANALYSIS:

 1 pass over DM reduced the maximum off-diagonal element to 0.85E-13

 Hybridization/Polarization Analysis of NLMOs in NAO Basis, Alpha Spin:
 NLMO / Occupancy / Percent from Parent NBO / Atomic Hybrid Contributions
 -------------------------------------------------------------------------------
   1. (1.00000)  99.9998% CR ( 1)Li  1
                         100.000% Li  1 s(100.00%)p 0.00(  0.00%)
   2. (1.00000) 100.0000% BD ( 1)Li  1- H  2
                           5.700% Li  1 s( 57.16%)p 0.75( 42.84%)
                          94.300%  H  2 s(100.00%)


 ***************************************************
 *******         Beta  spin orbitals         *******
 ***************************************************


 NATURAL POPULATIONS:  Natural atomic orbital occupancies

  NAO Atom No lang   Type(AO)    Occupancy      Energy
 -------------------------------------------------------
   1   Li  1  s      Cor( 1s)     1.00000      -2.72586
   2   Li  1  s      Val( 2s)     0.00000      -0.12055
   3   Li  1  px     Ryd( 2p)     0.00000       0.12726
   4   Li  1  py     Ryd( 2p)     0.00000      -0.09275
   5   Li  1  pz     Ryd( 2p)     0.00000      -0.09275

   6    H  2  s      Val( 1s)     0.00000      -0.06774


 Summary of Natural Population Analysis:

                                     Natural Population
             Natural    ---------------------------------------------
  Atom No    Charge        Core      Valence    Rydberg      Total
 --------------------------------------------------------------------
   Li  1    0.50000      1.00000     0.00000    0.00000     1.00000
    H  2    0.50000      0.00000     0.00000    0.00000     0.00000
 ====================================================================
 * Total *  1.00000      1.00000     0.00000    0.00000     1.00000

                                 Natural Population
 ---------------------------------------------------------
   Core                       1.00000 ( 99.9997% of    1)
   Valence                    0.00000 (  0.0000% of    0)
   Natural Minimal Basis      1.00000 (100.0000% of    1)
   Natural Rydberg Basis      0.00000 (  0.0000% of    1)
 ---------------------------------------------------------

    Atom No         Natural Electron Configuration
 ----------------------------------------------------------------------------
     Li  1      [core]
      H  2


 NATURAL BOND ORBITAL ANALYSIS, beta spin orbitals:

                            Occupancies       Lewis Structure    Low   High
         Max    Occ     -------------------  -----------------   occ   occ
  Cycle  Ctr   Thresh    Lewis   non-Lewis     CR  BD  nC  LP    (L)   (NL)
 ============================================================================
    1     2     0.95     1.00000   0.00000      1   0   0   0     0      0
 ----------------------------------------------------------------------------

 Structure accepted: No low occupancy Lewis orbitals

 The molecular units found in the alpha and beta manifolds are inequivalent.
 For labelling purposes, the molecular units of the beta system will be used.

 -------------------------------------------------------
   Core                      1.00000 (100.000% of   1)
  ==================      =============================
   Total Lewis               1.00000 (100.000% of   1)
  -----------------------------------------------------
   Valence non-Lewis         0.00000 (  0.000% of   1)
   Rydberg non-Lewis         0.00000 (  0.000% of   1)
  ==================      =============================
   Total non-Lewis           0.00000 (  0.000% of   1)
 -------------------------------------------------------


     (Occupancy)   Bond orbital / Coefficients / Hybrids
 ------------------ Lewis ------------------------------------------------------
   1. (1.00000) CR ( 1)Li  1            s(100.00%)
                                         1.0000  0.0000  0.0000  0.0000  0.0000
 ---------------- non-Lewis ----------------------------------------------------
   2. (0.00000) LV ( 1)Li  1            s( 62.19%)p 0.61( 37.81%)
   3. (0.00000) LV ( 1) H  2            s(100.00%)
   4. (0.00000) RY ( 1)Li  1            s(  0.00%)p 1.00(100.00%)
   5. (0.00000) RY ( 2)Li  1            s(  0.00%)p 1.00(100.00%)
   6. (0.00000) RY ( 3)Li  1            s( 37.81%)p 1.64( 62.19%)


 NHO DIRECTIONALITY AND BOND BENDING (deviation from line of nuclear centers at
                                      the position of maximum hybrid amplitude)

         [Thresholds for printing:  angular deviation  >  1.0 degree]
                                    p- or d-character  > 25.0%
                                    orbital occupancy  >  0.10e

                        Line of Centers        Hybrid 1             Hybrid 2
                        ---------------  -------------------  ------------------
           NBO            Theta   Phi    Theta   Phi    Dev   Theta   Phi    Dev
 ===============================================================================
    None exceeding thresholds


 SECOND ORDER PERTURBATION THEORY ANALYSIS OF FOCK MATRIX IN NBO BASIS

     Threshold for printing:   0.25 kcal/mol
    (Intermolecular threshold: 0.03 kcal/mol)
                                                          E(2) E(NL)-E(L) F(L,NL)
      Donor (L) NBO              Acceptor (NL) NBO      kcal/mol   a.u.    a.u.
 ===============================================================================

 within unit  1
       None above threshold

 from unit  1 to unit  2
       None above threshold

 from unit  2 to unit  1
       None above threshold

 within unit  2
       None above threshold


 NATURAL BOND ORBITALS (Summary):

                                                     Principal Delocalizations
           NBO                 Occupancy    Energy   (geminal,vicinal,remote)
 ===============================================================================
 Molecular unit  1  (Li)
 ------ Lewis --------------------------------------
    1. CR ( 1)Li  1             1.00000    -2.72586
 ------ non-Lewis ----------------------------------
    2. LV ( 1)Li  1             0.00000    -0.12338
    4. RY ( 1)Li  1             0.00000    -0.09275
    5. RY ( 2)Li  1             0.00000    -0.09275
    6. RY ( 3)Li  1             0.00000     0.13009
          -------------------------------
                 Total Lewis    1.00000  ( 99.9997%)
           Valence non-Lewis    0.00000  (  0.0000%)
           Rydberg non-Lewis    0.00000  (  0.0000%)
          -------------------------------
               Total unit  1    1.00000  (100.0000%)
              Charge unit  1    0.50000

 Molecular unit  2  (H)
 ------ Lewis --------------------------------------
 ------ non-Lewis ----------------------------------
    3. LV ( 1) H  2             0.00000    -0.06774
          -------------------------------
                 Total Lewis    0.00000  (  0.0000%)
           Valence non-Lewis    0.00000  (  0.0000%)
           Rydberg non-Lewis    0.00000  (  0.0000%)
          -------------------------------
               Total unit  2    0.00000  (100.0000%)
              Charge unit  2    0.50000

 $CHOOSE
  ALPHA
   BOND S 1 2 END
  END
  BETA
  END
 $END


 NATURAL LOCALIZED MOLECULAR ORBITAL (NLMO) ANALYSIS:

 1 pass over DM reduced the maximum off-diagonal element to 0.14E-11

 Hybridization/Polarization Analysis of NLMOs in NAO Basis, Beta Spin:
 NLMO / Occupancy / Percent from Parent NBO / Atomic Hybrid Contributions
 -------------------------------------------------------------------------------
   1. (1.00000)  99.9997% CR ( 1)Li  1
                         100.000% Li  1 s(100.00%)p 0.00(  0.00%)

 NORMAL TERMINATION of nbo program

 Maximum scratch memory used by NBO was 130817 words (1.00 MB)
 Maximum scratch memory used by GENNBO was 174 words (0.00 MB)
